dashboard/app/models/peer_review.rb

```diff
@@ -208,4 +209,8 @@ def self.get_potential_reviews(script, user)
       level_source_id: PeerReview.where(reviewer: user, script: script).pluck(:level_source_id)
     )
   end
+
+  def clear_data
+    update(data: SYSTEM_DELETED_DATA)
```

Comment:
Peer reviews are viewable by the submitter, so a submitter here would see 'system_deleted_data'

Why not just make this blank?